libs/components/src/lib/menu/menu.a11y.spec.ts

```diff
@@ -17,8 +18,10 @@ describe('a11y: vwc-menu', () => {
 		element.open = true;
 		element.ariaLabel = 'A11y label';
 		element.innerHTML = `
-      <div role="menuitem" id="id1">Menu Item 1</div>
-      <div role="menuitem" id="id2">Menu Item 2</div>
+      <vwc-menu-item text="Menu Item 1"></vwc-menu-item>
+      <vwc-menu-item control-type="checkbox" text="Menu Item Checkbox"></vwc-menu-item>
+      <vwc-didiver></vwc-didiver>
```

Comment:
There seems to be a typo:
```suggestion
      <vwc-divider></vwc-divider>
```